eth/handler.go

```diff
@@ -902,10 +924,40 @@ func (pm *ProtocolManager) GetValidatorPeers() []string {
 	return pm.server.ValPeers()
 }
 
+func (pm *ProtocolManager) GetSentryPeers() [][2]string {
+	sentryPeers := pm.server.SentryPeers()
+	returnArray := make([][2]string, len(sentryPeers))
+
+	for i, sp := range sentryPeers {
+		returnArray[i] = [2]string{sp.Node().String(), sp.ExternalNode.String()}
+	}
+
+	return returnArray
+}
+
+func (pm *ProtocolManager) Proxied() bool {
```

Comment:
--proxied would be a `miner`/`istanbul` option, so it should be a parameter in the config object for those packages.